packages/lerna/src/init.ts

```diff
@@ -26,7 +26,7 @@ export default async function (context: IContext, config: IPluginConfig): Promis
         context.version.new = lernaJson.version;
         publishConfig = lernaJson.command?.publish;
     } catch {
-        context.logger.warn(`Missing or invalid lerna.json in branch ${context.branch.name}`);
+        throw new Error(`Missing or invalid lerna.json in branch ${context.branch.name}`);
```

Comment:
Sounds like we are going to start requiring a lerna.json file?
Nothing wrong, just want to understand and discuss offline a bit about this (and other questions too)